root.tf

```diff
@@ -714,7 +780,7 @@ module "create_keycloak_users_s3_lambda" {
   vpc_id                         = module.shared_vpc.vpc_id
   lambda_create_keycloak_user_s3 = true
   private_subnet_ids             = module.backend_checks_efs.private_subnets
-  s3_bucket_arn                  = module.create_bulk_users_bucket.s3_bucket_arn
+  s3_bucket_arn                  = "arn:aws:s3:::tdr-create-bulk-keycloak-users-intg"
```

Comment:
Should the ARN have a hardcoded TDR environment `intg` at the end?

Reply:
No it shouldn't, no idea why I did this. I've put the module back.